sw/device/silicon_creator/manuf/base/provisioning_inputs.bzl

```diff
@@ -23,7 +23,7 @@ EARLGREY_SKUS = {
     # OTP Config: Emulation; DICE Certs: X.509; Additional Certs: None
     "emulation": {
         "otp": "em00",
-        "ca_data": "@//sw/device/silicon_creator/manuf/keys/fake:ca_data",
+        "ca_data": "@lowrisc_opentitan//sw/device/silicon_creator/manuf/keys/fake:ca_data",
```

Comment:
I'm sure there are many more uses of root-relative labels in the codebase that should be fixed (for example, the `dice_libs` immediately below this line).

Reply:
I think those might be fine? specifically, it was the `@//` that was causing problems, since that goes out to the root module, not just the root of this repo.

Doing a quick search, it seems like there is a few more of those though. If I globally replace them all, is the CI good enough to catch a potential regression?